src/containers/guest-home-page/how-it-works/mentorCardBoxArray.js

```diff
@@ -0,0 +1,27 @@
+import howItWorksMentorFirst from '~/assets/img/guest-home-page/howItWorksMentorFirst.svg'
+import howItWorksMentorSecond from '~/assets/img/guest-home-page/howItWorksMentorSecond.svg'
+import howItWorksMentorThird from '~/assets/img/guest-home-page/howItWorksMentorThird.svg'
+import howItWorksMentorFourth from '~/assets/img/guest-home-page/howItWorksMentorFourth.svg'
+
+export const mentorCardBoxArray = [

```

Comment:
why do you use 2 files for arrays with items
in my opinion it is better to put them in one file

Reply:
fixed